Changed the examples to use using for driver and session resources

zhenlineo opened the pull request:
Removed using driver if not needed
Changed driver to be a local var

Commit:
Changed the examples to use using for driver and session resources
Removed using driver if not needed
Changed driver to be a local var

lutovich commented:
@zhenlineo looks good to me. One question: `_driver` is now never disposed. Is this fine?

Commit:
Fix after code review
Moved driver management into Collection fixtures so that it will be closed after all ITs
